elements/pf-avatar/pf-avatar.css

```diff
@@ -1,3 +1,17 @@
+:host {
+  display: contents;
```

Comment:
setting `display: contents` causes the describe`without src attr` test to fail.  This is because an element set to display contents will always return 0 for offsetWidth or clientWidth as it doesn't produce a box. 

> These elements don't produce a specific box by themselves. They are replaced by their pseudo-box and their child boxes. Please note that the CSS Display Level 3 spec defines how the contents value should affect "unusual elements" — elements that aren't rendered purely by CSS box concepts such as replaced elements. See [Appendix B: Effects of display: contents on Unusual Elements](https://drafts.csswg.org/css-display/#unbox) for more details.

https://developer.mozilla.org/en-US/docs/Web/CSS/display#contents

Example: https://codepen.io/zeroedin/pen/ExrwgMR

```Suggestion
  display: inline-block;
```

Reply:
yeah more and more i'm thinking that we should avoid `:host { display: contents; }`, and maybe calculate a private css var in resize observer instead